types/chain.go

```diff
@@ -0,0 +1,8 @@
+package types
+
+var ChainId string
```

Comment:
Please set `ChainId` a field of `GreenfieldClient`.

Reply:
changed